x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -70,6 +78,15 @@ export class HeadlessChromiumDriver {
       interceptedCount = interceptedCount + (isData ? 0 : 1);
     });
 
+    // This really should never happen, but if we're tricked
+    // into loading anything from here something very fishy is going on
```

Comment:
puppeteer doesn't yet let you block responses, and since this can be a pretty bad situation, the job outright fails and closes down. I'm not sure if there's a better thing we can be doing here, but this is the best I can come up with given the API puppeteer currently has.